backend/app/cmd/server.go

```diff
@@ -775,11 +733,11 @@ func (s *ServerCommand) makeNotify(dataStore *service.DataStore, authenticator *
 			destinations = append(destinations, tg)
 		case "email":
 			emailParams := notify.EmailParams{
-				From:                s.Notify.Email.From,
```

Comment:
unless I'm blind i don't understand this "change", is it reformatting issue? Looks like some noise to me

Reply:
++, this shouldn't change.